WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/viewholders/ActivityViewHolder.kt

```diff
@@ -50,6 +51,23 @@ class ActivityViewHolder(val parent: ViewGroup) : BlockListItemViewHolder(
                 updateVisibility(item, parent.width / parent.context.resources.displayMetrics.density)
             }
         }
+
+        setupBlocksForAccessibility(item)
+    }
+
+    private fun setupBlocksForAccessibility(item: ActivityItem) {
+        val blocks = listOf(firstBlock, secondBlock, thirdBlock)
+
+        blocks.forEachIndexed { index, block ->
+            block.label.contentDescription = item.blocks[index].contentDescription
+
+            ContentDescriptionListAnnouncer(
```

Comment:
It feels a bit weird that we create an Object and we don't do anything with it. I know the class isn't testable anyway, but putting logic into constructor/init methods is a code smell. I'd consider moving the logic into a method. Wdyt?

Reply:
Resolved 962663d